pandas/io/formats/html.py

```diff
@@ -516,3 +469,50 @@ def _write_hierarchical_rows(self, fmt_values, indent):
                     row.insert(self.row_levels + self.fmt.tr_col_num, '...')
                 self.write_tr(row, indent, self.indent_delta, tags=None,
                               nindex_levels=frame.index.nlevels)
+
+
+class NotebookFormatter(HTMLFormatter):
+
```

Comment:
can you add a class doc-string / comment here and for ``HTMLFormatter`` to describe the differences.